docs/getting-started/products/yvaults/overview.md

```diff
@@ -10,17 +10,16 @@
 
 ## yVault Fee Structure
 
-**20% Performance Fee**: Deducted from yield earned every time a vault harvests a strategy.
+**Performance Fee**: Deducted from yield earned every time a vault harvests a strategy. 
 
-**2% Management Fee**: Flat rate taken from vault deposits over a year. The fee is extracted by minting new shares of the vault, thereby diluting vault participants. This is done at the time of harvest, and calculated based on time since the previous harvest.
+**Management Fee**: Flat rate taken from vault deposits over a year. The fee is extracted by minting new shares of the vault, thereby diluting vault participants. This is done at the time of harvest, and calculated based on time since the previous harvest.
 
-For example, a vault takes about .0055% of deposits per day on average (2 (percent)/365 (days)):
+The old structure used before [YIP-69](https://gov.yearn.finance/t/yip-69-reduce-and-cap-fees-through-yrates/12588) was 20% performance fees and 2% management fees, but this has changed and vaults now have a dynamic fee structure. Single asset vaults have no management fee. Fee values for all yVaults can be checked in real-time at https://yearn.watch/
```

Comment:
add a link about why it has changed https://gov.yearn.finance/t/yip-69-reduce-and-cap-fees-through-yrates/12588

Reply:
humm the link is there on the [YIP-69], do you mean like rename the link?